profiles/common/modules/features/cce_basic_config/cce_basic_config.features.filter.inc

```diff
@@ -137,6 +137,39 @@ function cce_basic_config_filter_default_formats() {
     ),
   );
 
+  // Exported format: Full HTML + Change tracking.
```

Comment:
I'm just wondering if we need this part. In the `cce_basic_config.install` file from line number 450 there is a solution which creates Full HTML + Change tracking profile based on Full HTML. For the moment I wasn't able to track any problems with this export so I think that it shouldn't have an impact.